templates/base/head.tmpl

```diff
@@ -7,20 +7,30 @@
 	<title>{{if .Title}}{{.Title}} - {{end}} {{if .Repository.Name}}{{.Repository.Name}} - {{end}}{{AppName}}</title>
 	<link rel="manifest" href="{{AppSubUrl}}/manifest.json" crossorigin="use-credentials">
 
+	{{if UseServiceWorker}}
 	<script>
 		if ('serviceWorker' in navigator) {
-  			window.addEventListener('load', function() {
```

Comment:
Does it no longer (or did it ever) matter that we wait for the `load` event?

Reply:
I removed it because it's unnecessary as ServiceWorker does not interact with the DOM, it should be available immediately.